UIP.py

```diff
@@ -1,10 +1,12 @@
 from scheduler import scheduler
 
 if __name__ == "__main__":
+
   try:
-    print("Press 1 to connect to internet or any other key to use this application in offline mode")
-    offline = int(input()) != 1
+    print("Hey this is UIP! you can use it to download images from reddit and also to schedule the setting of these images as your desktop wallpaper.\nPress 1 to connect to internet or any other key to use this application in offline mode")

```

Comment:
This is good enough, but generally it's preferrable to use two print statements rather than writing oneliner, or you could just use python multiline string with
```
print(''' Hello\n
This is a multiline string\n''');
```